src/fileFrom/UploadFromUrl.ts

```diff
@@ -53,10 +54,29 @@ export class UploadFromUrl extends UploadFrom {
   }
 
   private handleFromUrlStatusResponse = (token: string, response: FromUrlStatusResponse) => {
+    this.isFileUploadedFromUrlPolling = checkFileIsUploadedFromUrl({
```

Comment:
Further refactoring is required. There are two places, where response is checked: handleFromUrlStatusResponse and checkFileIsUploadedFromUrl. For me, pooler should be created unconditionally and make decisions about responses.

Reply:
Yes, you are right.